### ISSUE TITLE
Batch Renormalization: Towards Reducing Minibatch Dependence in Batch-Normalized Models

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#214]
## 一言でいうと

Batch Normalizationの改良版。バッチサイズが小さい時の問題、また推定時と学習時とで正規化方法に差異が出る問題を解消する試み。具体的には、最初は通常通りバッチ内で正規化するけど、徐々にデータ全体の正規化パラメーター(移動平均/分散)へシフトしていくという手法。

### 論文リンク

https://arxiv.org/abs/1702.03275

### 著者/所属機関

Sergey Ioffe
Google Inc
